adding new metapackages link

1297rohit opened the pull request:
Adding new links for metapackages.

- arcgis_dl_backbones - https://esri.maps.arcgis.com/home/item.html?id=2ee4a8305733432daeb084d36d2f70c6 (new link, old link needs to be updated)
- ArcGIS Timm Deep Learning Backbones Part 1 v1.0.0 - https://esri.maps.arcgis.com/home/item.html?id=fc123210a5fc4e3085df2d9fd04c9758
- ArcGIS Timm Deep Learning Backbones Part 2 v1.0.0 -  https://esri.maps.arcgis.com/home/item.html?id=5b379f6ed1af41d1b182c11e33552964
- ArcGIS Timm Deep Learning Backbones Part 3 v1.0.0 - https://esri.maps.arcgis.com/home/item.html?id=20012bcc8b8c4f4db824ab35a48530b0
- ArcGIS Timm Deep Learning Backbones Part 4 v1.0.0 - https://esri.maps.arcgis.com/home/item.html?id=659b266bf7b94b63aa2a1ea2b68f315e
- ArcGIS SAM Backbones 1.0.0 - https://esri.maps.arcgis.com/home/item.html?id=28054b9812344f058ba0e59aa1420cb7
- ArcGIS Mistral Backbone - https://esri.maps.arcgis.com/home/item.html?id=969d2fc57c834295af1a4a42cfd51f68
- ArcGIS Polygon Segmentation Postprocessing Backbone - https://esri.maps.arcgis.com/home/item.html?id=7e1485437b7b424597ea920271dab502

cleebp reviewed (approved):
Changes look good :+1:

ravinara001 commented:
@1297rohit  - we have [this](https://enterprise.arcgis.com/en/notebook/latest/administer/linux/extend-a-notebook-runtime.htm#ESRI_SECTION1_02A4CE532679454A97C219ED5F13F71C) documentation for ArcGIS Notebooks Server where the download link for DL backbone points to the older DL backbones item. Can you confirm if this the latest link that should be used for this?
https://esri.maps.arcgis.com/home/item.html?id=2ee4a8305733432daeb084d36d2f70c6

Also is the tar.gz still at version 2.0.1? => arcgis_dl_backbones-2.0.1-py_0.tar.bz2

1297rohit commented:
@ravinara001 Yes the latest link is this https://esri.maps.arcgis.com/home/item.html?id=2ee4a8305733432daeb084d36d2f70c6
and the tar.gz version is at 2.0.1 
Also we have updated the links and added link for ArcGIS Deep Learning Backbones for pytorch 1.4.0 in this PR https://github.com/Esri/deep-learning-frameworks/pull/86
